### DISCUSSION TITLE
How to aggregate function in flow query- Getting error

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#15299] [Archive]
![image](https://user-images.githubusercontent.com/36895131/187072631-fd41660e-03aa-43a3-9595-d540a425cef1.png)

getting this error

```json
{
  "status": 403,
  "code": "FORBIDDEN",
  "extensions": {}
}
```

**Comment by rijkvanzanten** [comment]
I think it's because the particular use of `IDs`. Try switching that to the raw editor with `{}` and make sure the value is `{{ $trigger.body.keys }}` without any quotes or other brackets around it
